cvat-ui/src/actions/users.actions.ts

```diff
@@ -0,0 +1,40 @@
+import { Dispatch, ActionCreator } from 'redux';
+
+
+export const getUsers = () => (dispatch: Dispatch) => {
+  dispatch({
+    type: 'GET_USERS',
+  });
+}
+
+export const getUsersSuccess = (users: [], isCurrentUser: boolean) => (dispatch: Dispatch) => {
+  dispatch({
+    type: 'GET_USERS_SUCCESS',
+    payload: users,
+    currentUser: isCurrentUser ? (users as any)[0] : isCurrentUser,
```

Comment:
Why do we get the first user as a current user if a filter wasn't specified?

Reply:
It is specified here https://github.com/opencv/cvat/pull/671/files/ec77c2a5f132d1e00eed3c3546227b6957f5e679#diff-a885ab78f1f9265e90fb7d64e1cdf54cR31

Reply:
@zankevich 
Got it, 
But other question. Maybe instead of isCurrentUser (which should be undefined or false) explicitly set 'undefined'?